src/board/generic_esp32/generic-esp32-board.c

```diff
@@ -13,6 +17,24 @@
 // ANSI
 #include <string.h>
 
+#define HZ_PER_KHZ  1000
+#define KHZ_PER_MHZ 1000
+#define HZ_PER_MHZ  (HZ_PER_KHZ * KHZ_PER_MHZ)
+
+#define SX1262_SPI_CLOCK_SPEED_MHZ  16
```

Comment:
is 16mhz the max speed? if so I'd make another define or constant to define that and then assign that to the clock speed.

Reply:
change the name of the define to include MAX for clarity